### ISSUE TITLE
Netlify demo deployment not working

### ISSUE THREAD

**Issue by jnoordsij** [sampotts/plyr#1734]
In merge request, an automatic deployment using Netlify is used. However, this demo deployment does not work properly when visited. This is probably caused by dependencies in the `demo` folder not being installed in the deployment process (Raven is missing).

### Expected behaviour
The deployment on netlify correctly runs the Plyr demo, so that it can be used to test browser support.

### Actual behaviour
Plyr gives a console error and browser built-in video player is used instead.

### Steps to reproduce
Visit https://plyr.netlify.com/ on any browser

### Environment

- Browser: Google Chrome
- Version: 80.0.3987.132 (64-bit)
- Operating System: Windows
- Version: 10

### Console errors (if any)
```
Uncaught ReferenceError: Raven is not defined
    at demo.js:15874
```

### Link to where the bug is happening
https://plyr.netlify.com/

**Comment by sampotts** [comment]
Yeah I've noticed this myself. The issue is the demo is a separate project to the library so it needs to run `npm install` in two directories, build the library, then build the demo. I started looking at it but ran out of time. I'd like to get it working so I can just host the demo on it and PR previews.

**Comment by jnoordsij** [comment]
Thanks for responding! It would be great indeed, would make testing and reviewing a lot easier.
